test/integration/System.t.sol

```diff
@@ -628,5 +604,48 @@ contract SystemIntegrationTest is Test {
 
         vm.prank(owners[0]);
         timelock.executeWhitelistedBatch(targets, values, calldatas);
+
+        bytes32 marketId = id(MarketParams(dai, ethenaUsd, oracle, irm, lltv));
+
+        Position memory position =
+            IMorpho(morphoBlue).position(marketId, address(timelock));
+
+        assertEq(position.supplyShares, 0, "incorrect supply shares");
+        assertEq(position.borrowShares, 0, "incorrect borrow shares");
+        assertEq(position.collateral, supplyAmount, "incorrect collateral");
+    }
+
+    /// ----------------- HELPERS -----------------
+
+    /// @notice The length of the data used to compute the id of a market.
+    /// @dev The length is 5 * 32 because `MarketParams` has 5 variables of 32 bytes each.
+    uint256 internal constant MARKET_PARAMS_BYTES_LENGTH = 5 * 32;
+
+    /// @notice Returns the id of the market `marketParams`.
+    function id(MarketParams memory marketParams)
+        internal
+        pure
+        returns (bytes32 marketParamsId)
+    {
+        assembly ("memory-safe") {
+            marketParamsId :=
+                keccak256(marketParams, MARKET_PARAMS_BYTES_LENGTH)
+        }
+    }
+
+    function signTxAllOwners(bytes32 transactionHash)
+        public
```

Comment:
If we intend to use these helper functions across other test files I believe we should create a base contract to this  and inherit here

If these helpers are only useful for integration tests, then I think we should make it private